dataland-frontend/tests/e2e/specs/company-metadata/SearchCompaniesForFrameworkDataDropdownFilter.ts

```diff
@@ -18,7 +18,9 @@ before(function () {
 describe("As a user, I expect the search functionality on the /companies page to adjust to the selected dropdown filters", () => {
   it("LKSG and SFDR should be displayed in the framework dropdown even though they are not yet implemented", () => {
     cy.ensureLoggedIn();
+    cy.intercept("**/api/companies/meta-information").as("companies-meta-information");
```

Comment:
Was hat das mit der Entfernung von Skyminder zu tun?

Reply:
Bei einem CI-Test gab es Timeout Errors in genau diesem Test. Meine Vermutung war, dass es eine Race-Condition ist - und dass sich die so verhindern lässt.

Der Bezug zu Skyminder ist also: Leider ist bei einem Skyminder-Testlauf dieser Test rot gewesen.

Reply:
alles klar